docs/guides/token_guide.md

```diff
@@ -31,3 +27,124 @@ Below is an overview of how the migration will take place:
+
+3. Via a **legacy Seed**. Use wallet library for generating, managing your new seed and migration process;
+
+
+
+#### Migration Guide - Via an Migration Address
+
+With this scenario, you generate your recovery phrase yourself. Then generate an address from it, generate migration address (as in the example below) and then use the same steps performed in section 1 for migrating.
+
+With this approach, its very easy to migrate funds to the new Chrysalis network. You just generate a new Chrysalis address and convert it to the old Trinary Format. 
+If you send Funds to this migration address, it bridges the network and you have the funds available in the Chrysalis network.
+
+#### generate migration address
+This is an example, how to create a Migration Address with Node.js:
+
+```javascript=
+function run() {
+  const { AccountManager } = require('../lib')
+  const manager = new AccountManager()
+  let address = "atoi1qpheapgshz9saf8mc6zdly96xyz5qdc2qspsv7ezem6fw8lv869ms3amw0g";
+  let migration_address = manager.generateMigrationAddress(address);
+  console.log(migration_address)
+}
+run()
+```
+
+Code: https://github.com/iotaledger/wallet.rs/blob/develop/bindings/nodejs/tests/migration.js
+
+
+### Be safe!
+
+Please take care that you...
+- ... dont create outputs that are lower that 1 Mi (to prevent Dust Protection).
+- ... you use a limited amount of inputs (<10) and create small Bundles (otherwise your PoW will just takes to long).
+- ... use the [bundle-miner](https://github.com/iotaledger/iota.rs/tree/migration/iota-bundle-miner), if old bundle hashes include spent address. The bundle-miner will create bundles that reveal as few new parts of the private key as possible.
+
+If you have spent addresses (which can be checked via the wereAddressesSpentFrom endpoint in the legacy network), we recommend to get the bundle hashes from the previous outgoing transactions for these addresses and use the bundle miner to create a bundle that reveals as few new parts of the private key as possible.
+
+More information about the migration bundle can be found in the [RFC-0035](https://github.com/luca-moser/protocol-rfcs/blob/rfc/wotsicide/text/0035-wotsicide/0035-wotsicide.md#migration-bundle).
+
+
+### Migration Guide via Seed
+The new recovery phrase and the migration address will be handled by the wallet library.
+
+Our IOTA Wallet Firefly is basically using wallet rs for the migration. There are three main methods exposed from wallet.rs:
+- [getMigrationData](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L203)
+- [createMigrationBundle](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L215)
+- [sendMigrationBundle](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L219)
```

Comment:
```suggestion
### Be safe!

Please take care that you...
- ... dont create outputs that are lower that 1 Mi (to prevent Dust Protection).
- ... use a limited amount of inputs (<10) and create small bundles (otherwise your PoW will just takes to long).
- ... use the [bundle-miner](https://github.com/iotaledger/iota.rs/tree/migration/iota-bundle-miner) if the old bundle hashes include spent address. The bundle-miner will create bundles that reveal as few new parts of the private key as possible.

If you have spent addresses (which can be checked via the wereAddressesSpentFrom endpoint in the legacy network), we recommend you get the bundle hashes from the previous outgoing transactions for these addresses and use the bundle miner to create a bundle that reveals as few new parts of the private key as possible.

More information about the migration bundle can be found in the [RFC-0035](https://github.com/luca-moser/protocol-rfcs/blob/rfc/wotsicide/text/0035-wotsicide/0035-wotsicide.md#migration-bundle).


### Migration Guide via Seed
The new recovery phrase and the migration address will be handled by the wallet library.

Our IOTA Wallet Firefly is basically using wallet rs for the migration. There are three main methods exposed from wallet.rs:
- [getMigrationData](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L203)
- [createMigrationBundle](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L215)
- [sendMigrationBundle](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L219)
```